core/src/com/cloud/storage/template/QCOW2Processor.java

```diff
@@ -75,6 +76,16 @@ public FormatInfo process(String templatePath, ImageFormat format, String templa
 
     @Override
     public long getVirtualSize(File file) throws IOException {
+        try {
+            long size = getTemplateVirtualSize(file);
```

Comment:
@karuturi Could you please use the QCOW2Utils.getVirtualSize() from the com.cloud.utils.storage package instead of using the function defined in the QCOW2Processor? :)